appinventor/appengine/src/com/google/appinventor/client/wizards/NewFolderWizard.java

```diff
@@ -58,6 +58,8 @@ public NewFolderWizard() {
     FolderTreeItem root = renderFolder(manager.getGlobalFolder());
     tree.addItem(root);
     tree.setSelectedItem(root);
+    addDialog.setWidth("400px");
+    addDialog.setAutoWidth(false); 
```

Comment:
This does not compile. setAutoWidth is not a recognized method.

Additionally, I think we want to make this change in our UIBinder templates. That's where most styling happens except for some legacy code.

You can see the different between the New Folder and New Project dialogs by comparing appinventor/client/wizards/NewFolderWizard.ui.xml to appinventor/client/wizards/youngandroid/NewYoungAndroidProjectWizard.ui.xml